ProjectFiles/Assets/Scripts/Network/Server.cs

```diff
@@ -17,34 +17,34 @@ public class Server
         private World world;
 
         private Dictionary<int, int> connectionPlayerIDs;
+
+        public ServerConfig Config { get; private set; }
+        private string IP => Config.IpAddress;
+        private ushort Port => Config.Port;
```

Comment:
What do these two things do? Is it like a symlink kinda thing?

Reply:
it's like a shorthand for a getter so you don't have to do Config.Port/Config.IpAddress. Here it is basically acting as a symlink

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members

Reply:
I'll just think of it as a symlink for now 🤷‍♂